lms/urls.py

```diff
@@ -1051,3 +1051,9 @@
 ]
 
 urlpatterns.extend(plugin_urls.get_patterns(plugin_constants.ProjectType.LMS))
+
+# Figures urls
+if 'figures' in settings.INSTALLED_APPS:
```

Comment:
Why we are adding this condition?

I believe we are going to install it anyway.

Reply:
removed it. It was mentioned in the setup guide but it works without it so you are right